### DISCUSSION TITLE
I get the following error on powershell 7.1.2 after upgrade from V2 to V3: UNABLE TO OPEN CONFIG

### DISCUSSION THREAD

**Discussion by ahrensmichael** [JanDeDobbeleer/oh-my-posh#494] [General]
Hi all

Do you know what I need to adjust?

Thanks in advance.
Michael

**Comment by JanDeDobbeleer** [comment]
Can you share your `$PROFILE`?

**Reply by ahrensmichael** [reply]
Yes, sure:

Import-Module posh-git
Import-Module oh-my-posh
Set-PoshPrompt -Theme powerline

**Reply by JanDeDobbeleer** [reply]
I'm going to give this a go but can add this to the top of your `$PROFILE` and try again?

`[console]::InputEncoding = [console]::OutputEncoding = New-Object System.Text.UTF8Encoding`

**Reply by ahrensmichael** [reply]
Hi Jan,

It works withe the following config in my $profile👍

Invoke-Expression (oh-my-posh --init --shell pwsh --config "$(scoop prefix oh-my-posh)/themes/jandedobbeleer.omp.json")

Thanks
Michael
